packages/cli/src/lib/snapshot/snapshot.ts

```diff
@@ -135,7 +136,7 @@ export class Snapshot {
         );
 
         if (isNotDone) {
-          throw new Error('Snapshot is still being processed');
+          throw new Error(this.operationGettingTooMuchTimeMessage());
```

Comment:
Is this an error though ? 🤔 

 My first reaction when I saw the message/screenshot, is that it's not bugged, it's "normal" that sometimes it takes a long time. 

So more like an "info" ?

Reply:
Related to that, what would be missing is something that explains: 

"After you run monitor command and it finally finishes, the next step would be to run XYZ".

Reply:
I guess this help message would only be shown within the context of a preview command. In this case, the next step would be to run `org:config:push`.

However, what would be the next step of a `org:config:push`?